Element.js

```diff
@@ -257,19 +257,29 @@ Element.prototype = {
 	 * @returns {Promise.<void>}
 	 */
 	type: function (value) {
-		function getPostData(arrayValue) {
-			if (self.session.capabilities.isWebDriver) {
-				return { value: arrayValue.join('').split('') };
+		function getPostData(arrayValue, paramName) {
+			var obj = {};
+			obj[paramName] = null;
+			if (self.session.capabilities.isWebDriver &&
+				!self.session.capabilities.typeParameterCalledText) {
```

Comment:
`valueParameterCalledText` or `valueParameterIsText` would be better since `value` is the actual wire protocol command.

Reply:
Fixed